_data/backup.yml

```diff
@@ -128,7 +128,7 @@ websites:
     img: mega.png
     tfa:
       - totp
-    doc: https://mega.nz/help/client/webclient/security-and-privacy/5bb5436cf1b70966038b456b
+    doc: https://mega.nz/help/client/ios/accounts-and-pro-accounts#how-do-i-enable-two-factor-authentication-on-my-account-5bb542b3f1b70961038b4571
```

Comment:
```suggestion
    doc: https://mega.nz/help/client/ios/5bb542b3f1b70961038b4571
```

Reply:
This link just redirects to the link i committed; the redirect might go away at some point, so i think for now it's better to have despite the greater character count.

Reply:
The new link points to the iOS help page, which is not so helpful for our purposes. The old link still works, and shows up-to-date information, as far as I can see. I'm not quite sure why this change is needed.

Reply:
I'm not sure it does work; I'm on Brave Browser and it just redirects to `https://mega.nz/help`... Feel free to close this if it's not a good suggestion.

Reply:
Okay, this is a bit strange. The page at https://mega.nz/help/client/webclient/security-and-privacy does look different depending on the browser, lacking a number of articles in some cases. This might be the reason that the original link is redirected to the main page for you.

I think this is definitely MEGAs fault, so I'm closing this PR.

Reply:
Sounds good, thanks.